### PR TITLE
DOC-4520 added PHP client page

### PR DESCRIPTION
**Pull request by andy-stark-redis**
[DOC-4520](https://redislabs.atlassian.net/browse/DOC-4520)

New client page for PHP. Note that we probably can't add examples for testing and we won't be supporting the full range of tabbed samples for PHP just yet.

[DOC-4520]: https://redislabs.atlassian.net/browse/DOC-4520?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/develop/connect/clients/php.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+- clients
+description: Connect your PHP application to a Redis database
+linkTitle: PHP
+title: PHP guide
+weight: 6
+---
+
+[`Predis`](https://github.com/predis/predis) is the recommended [PHP](https://php.net/)
+client for Redis. 
+The sections below explain how to install `predis` and connect your application to a Redis database.
```
Comment (dwdougherty) [inline content/develop/connect/clients/php.md:20]: For consistency...

```suggestion
The sections below explain how to install `Predis` and connect your application to a Redis database.
```

File: content/develop/connect/clients/php.md [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+{{< /note >}}
+
+`Predis` requires a running Redis or
+[Redis Stack]({{< relref "/operate/oss_and_stack/install/install-stack/" >}}) server.
+See [Getting started]({{< relref "/operate/oss_and_stack/install/" >}}) for Redis installation
+instructions.
+
+## Install
+
+Use [Composer](https://getcomposer.org/) to install the `predis` library
```
Comment (dwdougherty) [inline content/develop/connect/clients/php.md:33]: Choose one capitalization of predis and use throughout, unless there's a reason it's capitalized in some places, but not others.

```suggestion
Use [Composer](https://getcomposer.org/) to install the `Predis` library
```

File: content/develop/connect/clients/php.md [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+Use the following commands to generate the client certificate and private key:
+
+```bash
+openssl genrsa -out redis_user_private.key 2048
+openssl req -new -key redis_user_private.key -out redis_user.csr
+openssl x509 -req -days 365 -in redis_user.csr -signkey redis_user_private.key -out redis_user.crt
+```
+
+If you have the [Redis source folder](https://github.com/redis/redis) available,
+you can also use generate the certificate and private key with these commands:
```
Comment (dwdougherty) [inline content/develop/connect/clients/php.md:139]:
```suggestion
you can also generate the certificate and private key with these commands:
```

**Review by dwdougherty — CHANGES_REQUESTED** [review]
Needs a few tweaks.

**Comment by andy-stark-redis** [comment]
@dwdougherty Feedback applied. Thanks for spotting those problems.

**Review by dwdougherty — APPROVED** [review]
LGTM.

File: content/develop/connect/clients/php.md [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+
+```php
+<?php
+
+require 'vendor/autoload.php';
+
+use Predis\Client as PredisClient;
+
+$r = new PredisClient([
+                'scheme'   => 'tcp',
```
Comment (vladvildanov) [inline content/develop/connect/clients/php.md:53]: All of this are default values, can be omitted for simplicity

File: content/develop/connect/clients/php.md [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+                'port'     => 6379,
+                'password' => '',
+                'database' => 0,
+            ]);
+```
+
+Store and retrieve a simple string to test the connection:
+
+```php
+echo $r->set('foo', 'bar'), PHP_EOL;
```
Comment (vladvildanov) [inline content/develop/connect/clients/php.md:64]: PHP_EOL constant is equal to "\r\n", should be concatenated with other string `echo $r->set('foo', 'bar') . PHP_EOL;`

File: content/develop/connect/clients/php.md [hunk 6]
```diff
@@ -0,0 +1,314 @@
...
+
+```php
+$r->hset('user-session:123', 'name', 'John');
+$r->hset('user-session:123', 'surname', 'Smith');
+$r->hset('user-session:123', 'company', 'Redis');
+$r->hset('user-session:123', 'age', 29);
+
+echo var_export($r->hgetall('user-session:123')), PHP_EOL;
+/*
+array (
```
Comment (vladvildanov) [inline content/develop/connect/clients/php.md:82]: array() call is an old way to create an array, used before 5.4 version. The best practice now is a new notation, just like in the other versions `[  'name' => 'John', 'surname' => 'Smith', 'company' => 'Redis', 'age' => '29',];`

**Review by vladvildanov — CHANGES_REQUESTED** [review]

File: content/develop/connect/clients/php.md (hunk 5 above)
Comment (vladvildanov) [reply content/develop/connect/clients/php.md:58]: Hmm thanks for pointing out. In this case it should be fine. This was based on my personal experience, I never meet the comma approach.

File: content/develop/connect/clients/php.md (hunk 6 above)
Comment (andy-stark-redis) [reply content/develop/connect/clients/php.md:76]: @vladvildanov This comment shows the output from the echo command. The `var_export` function does actually format the array string like this (I've just tried it). In most of the other examples, we have ">>>" at the start of output sample comments to make it clearer that were showing the results of the previous command. I'd forgotten to add them here but I've put them in now.

File: content/develop/connect/clients/php.md (hunk 4 above)
Comment (andy-stark-redis) [reply content/develop/connect/clients/php.md]: @vladvildanov I've removed them. There's another example in the page that illustrates them, so as you say, best to use the simplest example to start off.

**Pull request merged**